src/Codeception/Coverage/Subscriber/Printer.php

```diff
@@ -21,6 +21,7 @@ class Printer implements EventSubscriberInterface
         'low_limit'         => '35',
         'high_limit'        => '70',
         'show_uncovered'    => false,
+        'add_uncovered'     => false,
```

Comment:
Please change default value to `true` to avoid breaking things for other users.
Default value of this property is set to `true` in https://github.com/sebastianbergmann/php-code-coverage/blob/8.0.1/src/CodeCoverage.php#L72

Reply:
Thank you! Done